tensorflow/core/kernels/random_op_cpu.h

```diff
@@ -85,7 +85,14 @@ struct FillPhiloxRandomTask<Distribution, false> {
                   int64_t start_group, int64_t limit_group, Distribution dist) {
     const int kGroupSize = Distribution::kResultElementCount;
 
-    gen.Skip(start_group);
+    // Decide skip strides according to different kResultElementCount:
+    // * `1 = (4 + 3) / 4` for normal Distribution.
+    // * `1 = (2 + 3) / 4` for double/int64 Distribution.
+    // * `4 = (16 + 3) / 4` for vectorized float/bfloat16 Distribution.
+    const int skip_strides =
+        (kGroupSize + gen.kResultElementCount - 1) / gen.kResultElementCount;
+    gen.Skip(start_group * skip_strides);
```

Comment:
Will this produce the same values as before at given elements?  We try to be consistent across platforms/backends (i.e. between CPU and XLA).

Reply:
Yes, the related UTs already checked same result across different platforms(CPU/GPU) and different iterations since I have fixed them before.